### ISSUE TITLE
Duplicated Symbol when you use Bolt API

### ISSUE THREAD

**Issue by yoon-boom** [aws/aws-sdk-ios#572]
I know AWS iOS uses the part of facebook Bolt library. When I tried to add Bolt library, it conflicts the symbol that are used in its own and from AWS library. Because extern string `BoltsFrameworkVersionString` conflicts, is there any workaround to solve this issue?

**Comment by minbi** [comment]
How are you using both libraries (Cocoapods, frameworks, etc.)?

You could temporarily remove the entry in `AWSBolts.h`. We will have fix released as soon as possible.

**Comment by yoon-boom** [comment]
Yes, I am using Cocoapods. Alternative workaround would be folk either Bolt or aws-sdk-ios then modify my own. But I would like to know how permanent build configuration through Podfile script or  xCode build setting instead folk and modify option.

**Comment by minbi** [comment]
I am using the following for Cocoapods and my project is able to compile. Is there something specific you are doing?

```
target 'TestProject' do
    pod 'AWSS3'
    pod 'Bolts'
end
```

**Comment by yoon-boom** [comment]
pod 'AWSCore'
pod 'AWSS3'
pod 'AWSCognito'
pod 'Bolts'

Dont have computer front of me but will let you know when I get back to home.

**Comment by yoon-boom** [comment]
`Installing AWSCore (2.5.0)
Installing AWSS3 (2.5.0)
Installing Bolts (1.8.4)`

Under CocoaPods version 1.1.1

I am guessing AWSCore->AWSBolts.h and Bolts.h has same extern string `extern NSString *const BoltsFrameworkVersionString;` causing that issue. So having AWSCore cause the build issue.

`duplicate symbol _BoltsFrameworkVersionString in:
    xxx/AWSCore/libAWSCore.a(AWSBolts.o)
    xxx/Bolts/libBolts.a(Bolts.o)
ld: 1 duplicate symbol for architecture i386
clang: error: linker command failed with exit code 1 (use -v to see invocation)`

**Comment by minbi** [comment]
The fix has been release in version 2.5.1
